assets/src/js/admin/plausible-admin.js

```diff
@@ -0,0 +1,126 @@
+!(function (e) {
+    var t = {};
+    function n(a) {
```

Comment:
@Ankur0812 Why do we need this file committed with minified version of source?

Reply:
We need to display Error message on save button.